include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -414,7 +414,7 @@ struct __parallel_transform_scan_single_group_submitter
     template <typename _KernelName, typename _Policy, typename _InRng, typename _OutRng, typename _InitType,
               typename _BinaryOperation, typename _UnaryOp>
     static auto
-    __launch_dynamic_bounds_scan(const _Policy& __policy, _InRng __in_rng, _OutRng __out_rng, ::std::size_t __n,
+    __launch_dynamic_bounds_scan(const _Policy& __policy, _InRng&& __in_rng, _OutRng&& __out_rng, ::std::size_t __n,
```

Comment:
Up to me - I don't understand why you are use now universal reference here, because there are no std::forward operations inside this function with params ```__in_rng``` and ```__out_rng```

Reply:
The forwarding references are here for two reasons:
1. Avoiding a copy of the range object if the calling side passes it as a glvalue
2. Consistency with the rest of the structs in this file (e.g., `__parallel_for_submitter`, `__parallel_partial_sort_submitter`, `__parallel_scan_submitter`)

Reply:
OK